bittensor/_neuron/text/core_validator/__init__.py

```diff
@@ -996,6 +996,7 @@ def forward(
                 neuron_stats.setdefault(_uid, {})
                 neuron_stats[_uid].update(_stats)  # gather neuron synapse validation measures and statistics
 
+        loss.backward()
```

Comment:
we're running a backward call during a forward? That's interesting!

Reply:
welp you caught me :p just now when i was double checking it, the backward is actually at somewhere else, so it is not needed here